Automattic/jetpack
# Search: Add MockUsageMeter along with DonutMeterContainer component

Fixes #26052

#### Changes proposed in this Pull Request:

* Introduce `DonutMeterContainer` component.
* Introduce `MockUsageMeter` to Jetpack Search Dashboard.
* Use pricing gate `isNewPricing202208` to display the `MockUsageMeter` component.

> Note by @dognose24: Since this DonutMeterContainer component may stay inside the Search package, I would prefer to complete the whole MockUsageMeter along with the DonutMeterContainer component to avoid redundant PR processes.

#### Other information:


#### Jetpack product discussion



Per discussion in Slack, we're going to build this directly in the Search package for now and look at Storybook integration as a follow up. It looks like we can build components as part of our package and still expose them to Storybook.

#### Does this pull request change what data or activity we track or use?



N/A

#### Testing instructions:






* Navigate the site URL with `/wp-admin/admin.php?page=jetpack-search&new_pricing_202208=1`.
* Ensure there will be a `UsageMeter` section displayed.
* Ensure the `UsageMeter` will be in line with the design.

| Before | After |
| --- | ----------- |
| <img width="1228" alt="截圖 2022-10-03 上午9 36 08" src="https://user-images.githubusercontent.com/6869813/193487221-7f84818f-9b44-46b0-9b11-5cb590b2211c.png"> | <img width="1280" alt="截圖 2022-10-03 上午9 35 39" src="https://user-images.githubusercontent.com/6869813/193487234-466edf71-6807-4849-a51c-3d9b5a18d29d.png"> |
